src/collective/isotope/browser/templates/isotope.pt

```diff
@@ -92,7 +92,7 @@ $(window).load(function(evt) {
                                          item_wf_state item/review_state;
                                          item_wf_state_class python:'state-' + view.normalizeString(item_wf_state);
                                          item_link python:item_type in view.use_view_action and item_url+'/view' or item_url;
-                                         item_has_image python:view.has_image(item);
+                                         item_has_image python:item.getIcon;
```

Comment:
Oh, awesome!  That means it's now a metadata column too, so no more waking up the object just to figure out if it has an image.  This is excellent news!